src/utils/get-functions.js

```diff
@@ -1,34 +1,25 @@
-const fs = require('fs')
-const path = require('path')
+const { listFunctions } = require('@netlify/zip-it-and-ship-it')
 
-const { findModuleDir, findHandler } = require('./finders')
+const { fileExistsAsync } = require('../lib/fs')
 
-module.exports = {
-  getFunctions(dir) {
-    const functions = {}
-    if (fs.existsSync(dir)) {
-      fs.readdirSync(dir).forEach((file) => {
-        if (dir === 'node_modules') {
-          return
-        }
-        const functionPath = path.resolve(path.join(dir, file))
-        const handlerPath = findHandler(functionPath)
-        if (!handlerPath) {
-          return
-        }
-        if (path.extname(functionPath) === '.js') {
-          functions[file.replace(/\.js$/, '')] = {
-            functionPath,
-            moduleDir: findModuleDir(functionPath),
-          }
-        } else if (fs.lstatSync(functionPath).isDirectory()) {
-          functions[file] = {
-            functionPath: handlerPath,
-            moduleDir: findModuleDir(functionPath),
-          }
-        }
-      })
-    }
-    return functions
-  },
+// List all Netlify Functions
+const getFunctions = async function (functionsSrcDir) {
+  if (!(await fileExistsAsync(functionsSrcDir))) {
+    return []
+  }
+
+  const functions = await listFunctions(functionsSrcDir)
```

Comment:
We need to add the function `name` to `listFunctions()` in `zip-it-and-ship-it` to make this work.

Reply:
Done in https://github.com/netlify/zip-it-and-ship-it/pull/231